Final Project Submission/hangman.cpp

```diff
@@ -0,0 +1,157 @@
+#include <iostream>
+#include <string>
+#include <cctype>
+#include <fstream>
+#include <algorithm>
+#include <cstdlib>
+#include <ctime>
+using namespace std;
+
+
+
+const string boardOne = " ------|\n |     |\n       |\n       |\n       |\n    -------\n\n";
+const string boardTwo = " ------|\n |     |\n 0     |\n       |\n        |\n    -------\n\n";
+const string boardThree = " ------|\n |     |\n 0     |\n |     |\n       |\n    -------\n\n";
+const string boardFour = " ------|\n |     |\n 0     |\n-|     |\n       |\n    -------\n\n";
+const string boardFive = " ------|\n |     |\n 0     |\n-|-    |\n       |\n    -------\n\n";
+const string boardSix = " ------|\n |     |\n 0     |\n-|-    |\n  \\    |\n    -------\n\n";
+const string boardSeven = " ------|\n |     |\n 0     |\n-|-    |\n/ \\    |\n    -------\n\n";
+
+int main()
+{
+  char game;
+  cout << "Play? y/n (lowercase) " << endl;
+  cin >> game;
+
+  while (game!='n')
+  {
+    srand(time(0));
+
+    const string wordList[15] = { "cat", "dog", "man", "leg", "fox", "hat", "eat", "log", "zap", "the", "fog", "can", "owl", "fin", "hot" };
+
+    string word = wordList[rand() % 15];
+
+
+    int wrongGuess = 0;
+    char letterGuess = 0;
+    int answer;
+    string checkWord = "***";
+
+
+    cout << "Time to guess a letter!" << endl;
+    cout << "\n" << boardOne << checkWord << endl;
+
+    while (wrongGuess != 6)
+    {
+      cout << "\nEnter a letter to guess: ";
+      cin >> letterGuess;
+      cout << "You guessed the letter: " << letterGuess << endl;
+
+      int position = word.find(letterGuess, 0);
```

Comment:
I'm mixed on this.  It's good you found how to use the `find()` function, but here you could have used a for loop over the string to find a letter - and replace it in multiple locations...

```
bool gotChar = false;
for (int charPos = 0; charPos < word.length(); charPos++) {
   if (word [charPos] == letterGuess) {
      gotChar = true;
      checkWord[charPos] = letterGuess;
  }
```